host/tests/dsClasses/dsVideoDevice_testConfig.yml

```diff
@@ -1,10 +1,36 @@
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
```

Comment:
Update the this file name in L3 documentation also

Reply:
Updated both the L3 test documents.